.github/workflows/pull-request.yml

```diff
@@ -52,18 +83,57 @@ jobs:
         with:
           npm-auth-token: ${{ secrets.NPM_TOKEN }}
 
+      - name: Cache Playwright browsers
+        if: ${{ success() || failure() }}
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-${{ needs.get-playwright-version.outputs.version }}
+
       - name: Install browsers
         run: npx playwright install --with-deps chromium firefox webkit
         shell: bash
 
-      - name: Run Playwright e2e tests
-        run: HOME=/root npm run e2e
+      - name: Run Playwright e2e tests with mocks
+        if: ${{ success() || failure() }}
```

Comment:
Same question here.

Reply:
This is required only for Test Steps so they will be marked as Failed but continue to next Test without failing whole job immediately
This is example how this look like: https://github.com/Backbase/golden-sample-app/actions/runs/14635571531/job/41065861168
Without that condition first test fail and will not run next step with tests
If we just add `continue-in-error: true` it will run ext test but step will be marked passed

Reply:
Yes because they are different types and not related Like responsiveness and accessibility

Reply:
But they are all E2E